xbmc/cores/VideoPlayer/Interface/Addon/InputStream.h

```diff
@@ -0,0 +1,15 @@
+/*
+ *  Copyright (C) 2017-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#define STREAM_PROPERTY_STREAMURL "streamurl" /*!< @brief the URL of the stream that should be played. */
```

Comment:
again, this is not a pvr thingy. streamurl is only used by pvr and meaningless for all other addons

Reply:
Ok, didn’t realise it was not used by other addons.